.github/workflows/build-gdal.sh

```diff
@@ -27,7 +27,7 @@ if [ -f CMakeLists.txt ]; then
 		-DCMAKE_BUILD_TYPE=Release \
 		-DBUILD_TESTING=OFF \
 		-DGDAL_USE_CURL=OFF \
```

Comment:
might also need to add `-DOGR_ENABLE_DRIVER_SQLITE=ON`

Reply:
@tbonfort if it keeps failing, I'll see if I can find a less finicky transactional driver to use for testing.

Reply:
build seems ok now. I'm not very keen on adding a spatialite dependency to the test-suite, are you able to rework your tests around that?

Reply:
@tbonfort yeah I think I should be able to get it to work without libspatialite-dev.